tests/testthat/test-logging.R

```diff
@@ -25,109 +25,6 @@ test_that("tune_log", {
   expect_snapshot(tune_log(ctrl_t, rs, task = "cube", type = "success"))
 })
 
-test_that("log issues", {
```

Comment:
There were a few differences in how we did things, which made these tests unnecessary or needlessly complicated to replicate. I'm deleting since these functions are tested by proxy later